### ISSUE TITLE
Add support for returning routing prefix of an IP

### ISSUE THREAD

**Issue by sarus** [runk/node-maxmind#68]
Hello,

I previously made use of the official MaxMind-DB-Reader-dotnet which had the ability to get the routing prefix of a looked up IP.  You can see discussion of the issue and resultant PR here:

https://github.com/maxmind/MaxMind-DB-Reader-dotnet/issues/22

Could this data also be added to the node-maxmind module?  The routing prefix is very useful for determining sets of IPs that have the same geolocation information.  

Thanks!

**Comment by runk** [comment]
Hey, it should be relatively straightforward to do. Would you mind crafting a PR? Not sure others also need it, otherwise I'd do it myself.

**Comment by sarus** [comment]
Hello,

No Problem.  I actually implemented this in a fork.

https://github.com/polarityio/node-maxmind/commit/904ff7bb26691b2b83fb88cad522a5b213b43438

I added a method `getWithRoutingPrefix ` to complement the existing `get` method and updated the `findAddressInTree` method to return an object containing both the routingPrefix and pointer (previously it just returned the single pointer value).

Does something like that work for you?
